lib/error.g

```diff
@@ -191,7 +191,15 @@ BIND_GLOBAL("ErrorInner",
         for x in earlyMessage do
             PrintTo("*errout*",x);
         od;
-        PrintTo("*errout*","\n");
+        if IsBound(IsRunningJupyterKernel) then
+            location := CURRENT_STATEMENT_LOCATION(context);
+            if location <> fail then
+                PrintTo(GAP_ERROR_STREAM, " at ", location[1], ":", location[2]);
```

Comment:
This causes the test suite to fail, as `GAP_ERROR_STREAM` is not defined. Perhaps you meant `"*errout*"`?

Reply:
he did mean `GAP_ERROR_STREAM`, because I used that global variable in my prototype code for error printing in the `JupyterKernel` package.

Whether it's useful to use `GAP_ERROR_STREAM` over `"*errout*"` is debatable, as long as it's search-and-replaceable, either is fine for me, with preference for `"*errout*"` because its consistent with the remainder of the codebase.

Reply:
Then this pull request is incomplete, as `GAP_ERROR_STREAM` is not defined. It cannot be merged until this is resolved, either by defining `GAP_ERROR_STREAM`, or by not using it.